istioctl/pkg/validate/validate_test.go

```diff
@@ -29,6 +29,114 @@ import (
 )
 
 const (
+	validDeploymentList = `
+apiVersion: v1
+items:
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    labels:
+      app: details
+      version: v1
+    name: details-v1
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: details
+        version: v1
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    labels:
+      app: hello
+      version: v2
+    name: hello
+  spec:
```

Comment:
@linsun suggests including the full Deployment .spec.  You can do that inline or you can copy one of our _samples_ into _istioctl/cmd/testdata_ and have the test load it from there.

Reply:
done